modules/rosa-sharing-vpc-hosted-zones.adoc

```diff
@@ -39,7 +39,10 @@ image::372_OpenShift_on_AWS_persona_worflows_0923_3.png[]
   ]
 }
 ----
-. Create a private hosted zone in the link:https://us-east-1.console.aws.amazon.com/route53/v2/[Route 53 section of the AWS console]. In the hosted zone configuration, the domain name is `<cluster_domain_prefix>.<reserved_dns_domain>`. The private hosted zone must be associated with the created VPC.
+. Create a private hosted zone in the link:https://us-east-1.console.aws.amazon.com/route53/v2/[Route 53 section of the AWS console]. Configure the hosted zone with the following details:
+
+* The domain name as `<cluster_domain_prefix>.<reserved_dns_domain>`
+* The private hosted zone must be associated with the created VPC.
```

Comment:
These aren't parallel - one is a fragment and one is a sentence, and the second one isn't a detail, it's more of a command. I would rewrite somehow to have them be parallel (per list guidance in IBM Style). Maybe something like:

```
Configure the hosted zone following this guidance:

* Express the domain name as `<cluster_domain_prefix>.<reserved_dns_domain>`.
* Associate the private hosted zone with the created VPC.
```